microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,637 @@
+
+* **New windows**: WebView2 instances on HoloLens 2 navigate within the same window by default, unlike on Desktop. Follow this default behavior for a better user experience.
+
+* **Enterprise authentication**: Automatic Single Sign-On (SSO) leveraging OS-level tokens is currently not supported in WebView2 on HoloLens 2. Users can still sign in by providing credentials, except for cases requiring device-level authentication. Cookie storage works as expected.
+
+* **User interactions**: Unlike native HoloLens 2 slates, WebView2 is best interacted with using far-interaction hand rays. Touch-to-swipe and scroll interactions might not be supported.
+
+* **Performance**: Complex websites with heavy use of JavaScript or advanced rendering may impact system performance or the host application's framerate. For general performance-related limitations and recommendations, see [Understanding performance for mixed reality](/windows/mixed-reality/develop/advanced-concepts/understanding-performance-for-mixed-reality) in the Mixed Reality documentation. Also see [Performance optimization](#performance-optimization), below.
+
+
+<!-- ------------------------------ -->
+#### Navigation
+
+In Step 6, we touched on some navigation methods. In this section, we'll expand on what we learned.
+
+
+<!-- ---------- -->
+###### IWebView interface
+
+<!-- todo: link to API Ref for IWebView
+* [IWebView::Load method]()
+-->
+
+The `IWebView` interface exposes a few methods, events, and properties related to page navigation. The main functionality exposed here is the ability to navigate to a given URL, by using `Load(Uri url)`:
+
+```C#
+public interface IWebView
+{
+    // Non-navigation methods are removed for clarity.
+
+    event WebView_OnNavigated Navigated;
+
+    Uri Page { get; }
+
+    void Load(Uri url);
+
+    void Reload(bool ignoreCache);
+}
+```
+
+
+<!-- ---------- -->
+###### IWithBrowserHistory interface
+
+The `IWithBrowserHistory` interface exposes a few methods and events related to page navigation. This mainly allows developers to navigate forward and backward, as you would expect with a typical web-browsing experience:
+
+```C#
+public interface IWithBrowserHistory : IWebView
+{
+    // Non-navigation methods are removed for clarity.
+
+    event WebView_OnCanGoForwardUpdated CanGoForwardUpdated;
+
+    event WebView_OnCanGoBackUpdated CanGoBackUpdated;
+
+    void GoBack();
+
+    void GoForward();
+}
+```
+
+
+<!-- ---------- -->
+###### SetVirtualHostNameToFolderMapping and SetVirtualHostMapping
+
+The `CoreWebView2.SetVirtualHostNameToFolderMapping` method enables mapping between a virtual host name and a folder path, making it accessible to websites using that host name. This method maps a local domain name to a local folder, so that the WebView2 control loads content from the specified local folder when attempting to access a resource for that domain.
+
+##### [.NET/C#](#tab/dotnetcsharp)
```

Comment:
@michaelfarnsworth is this article C# only?  if so, remove `#tab` lines and `---` terminator and just keep the first, .net link.

Reply:
Yeah, we should stick with C#.

Reply:
In fact, we may want to remove the API reference. Technically, that is the underlying API, but it's wrapped, instead of a true projection. Consequently, the API surface is slightly different. Not sure if there's a protocol to follow in that case.